app/routes/utils/get-status-update-options.js

```diff
@@ -0,0 +1,17 @@
+const { upperFirstLetter } = require('../../lib/display-helper')
+
+const statusUpdateOptions = {
+  IN_CHECK: 5,
```

Comment:
Was there an update on restricting just to these? Or has the move to withdraw been postponed to follow up ticket? And agreed/not agreed - these have been requested as manual updates previously?

Reply:
Withdrawn can only become the status for OW applications that have no claims against them. In that scenario you get the green button so override not needed.

Agreed/not agreed - was not required in the follow up meeting we had but I'll ask Indy to confirm this.

Reply:
ok let's proceed then